<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMultiReplace</class>
 <widget class="QDialog" name="DialogMultiReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace All in Multiple Documents</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Files in which to Replace All:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkShowPaths">
       <property name="text">
        <string>Show Path Names</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listFiles">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonReplace">
       <property name="text">
        <string>Replace</string>
       </property>
       <property name="icon">
        <iconset theme="edit-find-replace">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSelectAll">
       <property name="text">
        <string>Select All</string>
       </property>
       <property name="icon">
        <iconset theme="edit-select-all">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDeselectAll">
       <property name="text">
        <string>Deselect All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-cancel">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogMultiReplace</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
